examples/language-sdk-instrumentation/golang-push/rideshare-alloy/config.alloy

```diff
@@ -3,6 +3,40 @@ pyroscope.receive_http "default" {
         listen_address = "0.0.0.0"
         listen_port = 9999
     }
+    forward_to = [pyroscope.relabel.filter_profiles.receiver]
+}
+
+pyroscope.relabel "filter_profiles" {
+    // Convert region to datacenter
+    rule {
+        action = "replace"
+        source_labels = ["region"]
+        target_label = "datacenter"
+    }
+
+    // Add environment label
+    rule {
+        action = "replace"
+        target_label = "environment"
+        replacement = "demo"
+    }
+
+    // Add component label based on application name
+    rule {
+        action = "replace"
+        source_labels = ["__name__"]
```

Comment:
`__name__` is a profile name like `process_cpu` or `memory` . I am not understanding why is it compared against `ride-sharing-app.*`

Reply:
for history: now I udnerstand why `__name__` is used. This is because `__name__` is the special lable in OG flameql.

Reply:
I think we should somehow translate `__name__` to `service_name` transparentyl

Reply:
I'll change it to avoid confusion here 👍 On the other hand, I've just opened a slack thread to decided what's the best path to follow here.